### DISCUSSION TITLE
Inconsistent with the A1111 SD generation result？

### DISCUSSION THREAD

**Discussion by tsjslgy** [lllyasviel/stable-diffusion-webui-forge#111] [General]
Used my own LORA, sent PNG information directly, same Prompt and seed, all the same but the results were different, and the quality was very low.
Tried several times with the same result, wondering if it was really the same thing.

**Comment by lllyasviel** [comment]
there was some similar reports like https://github.com/lllyasviel/stable-diffusion-webui-forge/issues/85
but they are mostly caused by users forgetting to turn on some features that are deeply hidden in their original A1111 settings, like code former, etc.
usually the difference can be resolved after some investigation of all old/new settings

**Reply by tsjslgy** [reply]
sdwebui
![1](https://github.com/lllyasviel/stable-diffusion-webui-forge/assets/56958241/ffac9295-8acb-4213-85cf-e19d033a3a2a)
forge
![2](https://github.com/lllyasviel/stable-diffusion-webui-forge/assets/56958241/99c27b76-b8b4-4c08-bacf-24c45b6dc5b6)
seed:12345  step:20   

a1111 webui
![tmp3dnasgni](https://github.com/lllyasviel/stable-diffusion-webui-forge/assets/56958241/d31d5fa4-84d1-4853-9f32-8fc92933e335)
![tmpieb6e_uq](https://github.com/lllyasviel/stable-diffusion-webui-forge/assets/56958241/13f4c703-269f-40cc-984f-2774057db16e)
![tmpvh9m599v](https://github.com/lllyasviel/stable-diffusion-webui-forge/assets/56958241/886c8c29-138d-4fea-8fef-2430b68894f0)
![tmp6q8w3psz](https://github.com/lllyasviel/stable-diffusion-webui-forge/assets/56958241/05233d46-6b9a-41f6-9076-7146f517536a)
forge
![tmp9_19uktx](https://github.com/lllyasviel/stable-diffusion-webui-forge/assets/56958241/a0bf03f6-9da7-4395-889f-9153f47d0254)
![tmp6qrwqyhr](https://github.com/lllyasviel/stable-diffusion-webui-forge/assets/56958241/b0a7308f-e10c-4ae4-93e5-709df48775ea)
![tmpb4sbfjvi](https://github.com/lllyasviel/stable-diffusion-webui-forge/assets/56958241/da20b6cf-f9c6-4c92-9006-6c6a1c059aa4)
![tmp_w3pj0w9](https://github.com/lllyasviel/stable-diffusion-webui-forge/assets/56958241/c59c8115-5c6b-4ae3-9c87-3000b3b1a77b)

**Reply by tsjslgy** [reply]
This is with seed 12345, the difference would be more noticeable when generating higher quality images.

**Reply by CCpt5** [reply]
You should go into settings for both A1111 and Forge and open up Sys Info (tab) in settings. Pick "Open as txt in new page". Do that on both A1111 and Forge at the same time and just go between tabs making sure all of the values you have set for A1111 are the same in forge. As mentioned there are a few optimizations that will make SD non-deterministic w/ generations (XFormers for example can).....probably something there causing a difference.

The other thing that probably isn't it, but there's a bug at the moment (#51) where the preview/live-preview does not display at full resolution even if it's set as full in settings. If you're watching and judging quality of generations by the preview while rendering (before the batch completely finishes) that could be why you're seeing bad quality. I reported this issue the other day here: https://github.com/lllyasviel/stable-diffusion-webui-forge/issues/51  - The outputted files and final preview post-batch should be correct/full quality though.
